components/Layout/index.tsx

```diff
@@ -133,7 +142,6 @@ function LayoutSearch(props: LayoutProps) {
                         </>
                     )}
                 <Footer />
-                {showZilpay && <ZilPay />}
```

Comment:
Then where are we running this component? We could call the needed function instead.

Reply:
We already show the component on Dashboard.tsx, if we call again it'll causing duplication and infinity loop